book.rb

```diff
@@ -0,0 +1,15 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
```

Comment:
- [ ] There should be getter and setter (`attr_accessor`) for **:rentals**. Kindly add this.